blockmanager.go

```diff
@@ -2620,6 +2620,13 @@ func (b *blockManager) handleHeadersMsg(hmsg *headersMsg) {
 					hmsg.peer.Disconnect()
 					return
 				}
+				// Make sure headers being processed are connecting
+				prevBlockHashInReorganizationList := b.reorgList.Back().Header.BlockHash()
+				if !reorgHeader.PrevBlock.IsEqual(&prevBlockHashInReorganizationList) {
```

Comment:
We try to keep lines at 80 characters max length (using 8 spaces for tabs). Maybe choose a shorter name?

Reply:
Could you assist?  I agree and many thanks for the review.